### PR TITLE
chore: add more information about generating docs

### LINKED ISSUE #697: [Question] Generating JSON / YAML
## Is your feature request related to a problem? Please describe.
In README file => Global Metadata, i think it has a missunderstanding (can be me, idk).
I noticed that we can generate json / yaml at the same time by using 2 objects in spec/swagger_helper.rb.

But based on [PR 251](https://github.com/rswag/rswag/pull/251/files), if you set the `config.swagger_format`, you will change all to json OR yaml.

## Describe the solution you'd like
I created a PR, that says that you can use:
`config.swagger_format = :yaml`

But, to be honest, i want to know if is really possible to create `json` and `yaml` at the time.
if not, consider reviewing my contribution on `Readme.md` file

## What support could we give you, so you could implement this yourself?
Yes, it's just a update on docs

## Additional context
image

### PR DESCRIPTION
**Pull request by alexcastrodev**
## Problem
It's a missunderstanding of config

## Solution
improvements of docs

### The changes I made are compatible with:
- [x] OAS2
- [x] OAS3
- [x] OAS3.1

### Related Issues
https://github.com/rswag/rswag/issues/697

### Checklist
- [ ] Added tests
- [x] Changelog updated
- [x] Added documentation to README.md
- [ ] Added example of using the enhancement into [test-app](https://github.com/rswag/rswag/tree/master/test-app)

### Steps to Test or Reproduce
Not needed

### CODE REVIEW

**Comment by alexcastrodev** [comment]
@romanblanco i have been updated based on latest version

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
 
 - Add warning about methods renaming (https://github.com/rswag/rswag/pull/688)
+- Added more details about config swagger format [#698](https://github.com/rswag/rswag/pull/698)
```
Comment (romanblanco) [inline CHANGELOG.md]: @AlexcastroDev, this should be added under the `[Unreleased]` tag.

**Comment by romanblanco** [comment]
Thank you for this, @AlexcastroDev!

Minor adjustment needed, than it's good to go :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (alexcastrodev) [reply CHANGELOG.md]: Good catch ! Thanks

**Pull request merged**